kbs/src/api_server.rs

```diff
@@ -100,7 +101,14 @@ impl ApiServer {
             self.config.http_server.sockets
         );
 
-        let http_config = self.config.http_server.clone();
+        let mut http_config = self.config.http_server.clone();
+
+        // TODO: make the address/port configurable?
+        http_config.sockets.push(SocketAddr::new(
+            IpAddr::V4(Ipv4Addr::new(0, 0, 0, 0)),
+            12345,
+        ));
```

Comment:
why do we listen on another port? a /metrics route just sits alongside the other routes (like /health), no?

Reply:
Are we thinking that this endpoint is trusted? The KBS endpoint is generally exposed to the internet so it could make sense to use another one if we don't want to expose things.

Reply:
if such a service is exposed to the internet it would usually sit behind an L7 proxy/api gw, which doesn't expose such routes

Reply:
Same. There is no point in exposing metrics to the public by default.

Reply:
It's currently served on the same port as the KBS protocol as per this review request.  It's easy enough to change in the future anyway, should we make that call.